{"name": "test2", "status": "passed", "description": "Test 2: Wrong input gcal caught by parameter checking (exception thrown)\n        ", "start": 1758043979802, "stop": 1758043980024, "uuid": "5a9fe248-1ddd-4e30-8347-38e3b47b1807", "historyId": "ee29d264a15b2947958431811cb882a7", "testCaseId": "ee29d264a15b2947958431811cb882a7", "fullName": "casatasks.test_task_smoothcal.smoothcal_test#test2", "labels": [{"name": "parentSuite", "value": "casatasks"}, {"name": "suite", "value": "test_task_smoothcal"}, {"name": "subSuite", "value": "smoothcal_test"}, {"name": "host", "value": "casa-rockylinux8-amd64-perf07"}, {"name": "thread", "value": "597558-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "casatasks.test_task_smoothcal"}], "titlePath": ["casatasks", "test_task_smoothcal.py", "smoothcal_test"]}